README.md

```diff
@@ -11,6 +11,135 @@ supported boards and instructions on how to setup environment and build images.
 
 ![Mender logo](https://mender.io/user/pages/05.resources/06.digital-assets/logo.png)
 
+## Structure
+
+This section is for anyone interested in contribution support for an board.
+
+meta-mender-community is a top level layer which is not actually a layer it
```

Comment:
meta-mender-community is a repository containing multiple Yocto layers for integrating Mender onto various boards.